### PR TITLE
IEN-626 | hide filter tab on manage applicants page

### PR DESCRIPTION
**Pull request by sunghwan-park-fw**

### CODE REVIEW

File: apps/web/src/components/user/AclMask.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { PropsWithChildren, ReactNode } from 'react';
 import { Access, hasAccess } from '@ien/common';
 import { useAuthContext } from '../AuthContexts';
+import * as React from 'react';
```
Comment (coryatwater-FW) [inline apps/web/src/components/user/AclMask.tsx:4]: do we need this line?

File: apps/web/src/utils/is-ha-users.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Authorities, EmployeeRO, RoleSlug } from '@ien/common';
+
+export const isHaUsers = (user?: EmployeeRO): boolean => {
+  return (
+    !!user &&
+    user.roles.every(role => role.slug !== RoleSlug.Admin) &&
+    user.organization !== Authorities.HMBC.name
```
Comment (coryatwater-FW) [inline apps/web/src/utils/is-ha-users.ts:7]: what if the user is MoH?

**Review by coryatwater-FW — APPROVED** [review]
couple comments, but looks good to me

File: apps/web/src/utils/is-ha-users.ts (hunk 2 above)
Comment (sunghwan-park-fw) [reply apps/web/src/utils/is-ha-users.ts:7]: MoH has only reporting role by default so then don't have higher privileges than HA.

File: apps/web/src/components/user/AclMask.tsx (hunk 1 above)
Comment (sunghwan-park-fw) [reply apps/web/src/components/user/AclMask.tsx]: It's why we shouldn't do `copy and paste`

**Pull request merged**